react-native/react/actions/tracker.js

```diff
@@ -18,6 +18,55 @@ type TrackerActionCreator = (dispatch: Dispatch, getState: () => {tracker: RootT
 
 // TODO make actions for all the call back stuff.
 
+export function startTimer (): (dispatch: Dispatch, getState: any) => void {
```

Comment:
Use the `TrackerActionCreator` type here. It's more descriptive about the `getState` argument.